flowmachine/flowmachine/features/subscriber/displacement.py

```diff
@@ -97,8 +97,8 @@ def __init__(
         self,
         start,
         stop,
-        modal_locations=None,
-        statistic="avg",
+        reference_location,
+        value="avg",
```

Comment:
See comment above, I'd change this back to `statistic` (and similarly below, except the column name in the SQL query in L193).